mass_mailing_custom_unsubscribe/static/src/js/unsubscribe.js

```diff
@@ -0,0 +1,274 @@
+                        $('#info_state').removeClass('invisible');
+                        $info_state.removeClass('alert-success').removeClass('alert-info').removeClass('alert-warning').addClass('alert-error');
+                    }
+                    $('#button_add_blacklist').hide();
+                    $('#button_remove_blacklist').show();
+                    $('#unsubscribed_info').hide();
+                }
+            })
+            .guardedCatch(function () {
+                $('#subscription_info').html(_t('An error occured. Please try again later or contact us.'));
+                $info_state.removeClass('alert-success').removeClass('alert-info').removeClass('alert-warning').addClass('alert-error');
+            });
+    });
+
+    $('#button_remove_blacklist').click(function (e) {
+        e.preventDefault();
+
+        ajax.jsonRpc('/mailing/blacklist/remove', 'call', {'email': email, 'mailing_id': mailing_id, 'res_id': res_id, 'token': token})
+            .then(function (result) {
+                if (result == 'unauthorized'){
+                    $('#subscription_info').html(_t('You are not authorized to do this!'));
+                    $('#info_state').removeClass('invisible');
+                    $info_state.removeClass('alert-success').removeClass('alert-info').removeClass('alert-error').addClass('alert-warning');
+                }
+                else
+                {
+                    if (result) {
+                        $('#subscription_info').html(_t("You have been successfully <strong>removed from our blacklist</strong>. "
+                                + "You are now able to be contacted by our services."));
+                                $('#info_state').removeClass('invisible');
+                        $info_state.removeClass('alert-warning').removeClass('alert-info').removeClass('alert-error').addClass('alert-success');
+                        toggle_opt_out_section(true);
+                    }
+                    else {
+                        $('#subscription_info').html(_t('An error occured. Please try again later or contact us.'));
+                        $('#info_state').removeClass('invisible');
+                        $info_state.removeClass('alert-success').removeClass('alert-info').removeClass('alert-warning').addClass('alert-error');
+                    }
+                    $('#button_add_blacklist').show();
+                    $('#button_remove_blacklist').hide();
+                    $('#unsubscribed_info').hide();
+                }
+            })
+            .guardedCatch(function () {
+                $('#subscription_info').html(_t('An error occured. Please try again later or contact us.'));
+                $('#info_state').removeClass('invisible');
+                $info_state.removeClass('alert-success').removeClass('alert-info').removeClass('alert-warning').addClass('alert-error');
```

Comment:
```suggestion
                $info_state.removeClass('alert-success alert-info alert-warning').addClass('alert-error');
```
**Check other places with similar "call chain"...